packages/material-ui/src/Button/Button.js

```diff
@@ -445,7 +448,10 @@ Button.propTypes = {
    * `small` is equivalent to the dense button styling.
    * @default 'medium'
    */
-  size: PropTypes.oneOf(['large', 'medium', 'small']),
+  size: PropTypes /* @typescript-to-proptypes-ignore */.oneOfType([
+    PropTypes.oneOf(['large', 'medium', 'small']),
```

Comment:
It might make more sense to list them in this order

```suggestion
    PropTypes.oneOf(['small', 'medium', 'large']),
```